src/Datagrid/ProxyQuery.php

```diff
@@ -56,7 +56,7 @@ public function execute(array $params = [], $hydrationMode = null)
             $queryBuilder->sort($sortBy, $this->getSortOrder());
         }
 
-        return $queryBuilder->getQuery()->execute($params, $hydrationMode);
+        return $queryBuilder->getQuery()->execute();
```

Comment:
This is right? This method does not use the passed parameters anymore

Reply:
It was found by phpstan, [in the the `Query::execute` method of the `doctrine/mongo-odm` class does not have parameters](https://github.com/doctrine/mongodb-odm/blob/1.3.x/lib/Doctrine/ODM/MongoDB/Query/Query.php#L301).

Reply:
Should we throw a deprecation warning / error in that case?

Reply:
But the method `execute` ([from the Query class of doctrine/mongo-odm](https://github.com/doctrine/mongodb-odm/blob/1.3.x/lib/Doctrine/ODM/MongoDB/Query/Query.php#L301)) never had those parameters.

I mean, this call is not to `ProxyQuery::execute`.

Reply:
> But the method `execute` ([from the Query class of doctrine/mongo-odm](https://github.com/doctrine/mongodb-odm/blob/1.3.x/lib/Doctrine/ODM/MongoDB/Query/Query.php#L301)) never had those parameters.
> 

That's right, but if you pass a parameter to this method on purpose and they are completely ignored, the user should be informed

Reply:
Can you please help with the message?

Reply:
>Passing arguments to Query::execute() is deprecated since sonsata-project/doctrine-mongodb-admin-bundle 3.x. They are ignored.

Reply:
Thanks! I'll continue this PR once https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/368 is merged.